apps/E2E/README.md

```diff
@@ -69,6 +69,25 @@ _Note: It could take up to a minute to load the test app with WebDriverIO, don't
 
 _Note: It could take up to a minute to load the test app with WebDriverIO, don't panic, the tests will run :)_
 
+## iOS Steps
+
+First check that iOS configs in wdio.conf.ios.js are updated to match your dev environment. If the config doesn't match you may get an error message "App with bundle identifier 'com.microsoft.ReactTestApp' unknown"
+
+- 'appium:deviceName' should be updated to a simulator that you have. You can check the ones you have by going to the 'Devices and Simulators' window in Xcode.
+- 'appium:platformVersion' should be updated to match the iOS version of the simulator you're using. You can check what version this should be in the same 'Devices and Simulators' window.
+
+1. Follow step #1 from "Win32 Steps" section above.
+2. POD Install
+   - C:\repo\fluentui-react-native> `cd apps\fluent-tester\ios`
```

Comment:
"C:\repo\fluentui-react-native> " just curious. is this some sort of window-ism we are inheriting from the e2e test tool?